Dockerfile

```diff
@@ -91,6 +98,9 @@ RUN git clone --recursive ${YARA_URL} yara-python && \
     rm -fr /tmp/yara-python && \
     sed -i 's/APKID_ENABLED.*/APKID_ENABLED = True/' /root/Mobile-Security-Framework-MobSF/MobSF/settings.py
 
+# Set Java 11 Directory
+RUN sed -i 's#JAVA_DIRECTORY = ""#JAVA_DIRECTORY = "/usr/bin/"#' /root/Mobile-Security-Framework-MobSF/MobSF/settings.py
```

Comment:
Is this required. Wouldn't Java 11 be the default. Instead of this. Make sure Java 11 is the default.

Reply:
You should investigate why it's failing.

Reply:
it come's from the fact that script in profile.d are not launched , so the jdk.sh script which set the ENV is not sourced , shell in docker is not interactive

Reply:
@superpoussin22 Can we avoid setting Java directory as in line 102?

Reply:
@ajinabraham yep done 
I modify utils.py cause /usr/bin/python did note provide Oracle in output 
let me know if ok for you